src/control/cmd/agent/main.go

```diff
@@ -51,7 +51,7 @@ const (
 type cliOptions struct {
 	AllowProxy bool       `long:"allow-proxy" description:"Allow proxy configuration via environment"`
 	Debug      bool       `short:"d" long:"debug" description:"Enable debug output"`
-	JSON       bool       `short:"j" long:"json" description:"Enable JSON output"`
+	JSONLog    bool       `short:"J" long:"json-logging" description:"Enable JSON output"`
```

Comment:
Curious about the change in name. It's not really logging changes, is it? It seems below that it's still stdout output, just formatted as JSON.

Reply:
maybe this is called logging because the agent is not interactive and so the main output is logging

Reply:
The name change for this option in the agent and server is for consistency with dmg. In dmg, the `--json` option now results in results being dumped to stdout. The `--json-logging` option configures the logger to write [json-formatted log messages](https://github.com/daos-stack/daos/blob/master/src/control/logging/logging_test.go#L147), e.g.

```json
{"level":"DEBUG", "time":"2020-04-13...", "message":"foo"}
```

Reply:
OK, fine with me.

Reply:
Ah, I see a potential source of confusion. I didn't update the option description as I did in the dmg. Will do that.